__init__.py

```diff
@@ -33,10 +41,10 @@
 To run some script in the context of the code, one should temporarily alter the `PYTHONPATH`\
 environment variable:
 
-    PYTHONPATH=. python3 some_script.py
+    PYTHONPATH=. python3.5 some_script.py
 
 For example, to run the cosmic neutrino background temperature test:
 
-    PYTHONPATH=. python3 tests/cosmic_neutrino_temperature
+    PYTHONPATH=. python3.5 tests/cosmic_neutrino_temperature
```

Comment:
What about this 3.5?

Reply:
The command python3 won't necessarily run python3.5

Reply:
And `python3.5` definitely won't run Python 3.6. And there's no such command on a system where Python 3.6 is the default interpreter

Reply:
I added "using for example Python 3.5" in the current one.